pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -120,6 +127,11 @@ func (p *criStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 			continue
 		}
 
+		caStats, caFound := caInfos[containerID]
```

Comment:
nit: Move this below, it's weird to split the `!caFound` case and `caFound` case.